src/parser.ml

```diff
@@ -223,7 +223,8 @@ let parser any_ident =
   | id:escaped_ident -> id
 
 (** Identifier (regular and non-keyword, or escaped). *)
-let parser ident = id:any_ident -> in_pos _loc id
+let parser ident = 
+  | id:any_ident -> in_pos _loc id
```

Comment:
Do not introduce changes not related to the PR.

Reply:
Please, remove this change.

Reply:
Done.

Reply:
I don't see anything done here. Did you push your changes?